inventory/sample/hosts.ini

```diff
@@ -1,6 +1,7 @@
 # ## Configure 'ip' variable to bind kubernetes services on a
 # ## different ip than the default iface
 # ## We should set etcd_member_name for etcd cluster. The node that is not a etcd member do not need to set the value, or can set the empty string value.
+# [all]
```

Comment:
The [all] is optional, but that's fine.
 
Could you uncomment it? as well as all other groups ? 
```
[all]
# node1
# node2
....
[kube-master]
#node1
...
[kube-node]
# node1
[etcd]
....
```

Reply:
Done